packages/loopring_v3/contracts/impl/libexchange/ExchangeAdmins.sol

```diff
@@ -45,6 +51,29 @@ library ExchangeAdmins
         );
     }
 
+    function setMaxAgeDepositUntilWithdrawable(
+        ExchangeData.State storage S,
+        uint32 newValue
+        )
+        external
+        returns (uint32 oldValue)
+    {
+        require(!S.isInWithdrawalMode(), "INVALID_MODE");
+        require(
+            newValue >= 3 hours ||
```

Comment:
Any reason for the lower limit? Even 3 hours is pretty long...

Reply:
change it to > 0